src/Estimators/EnergyDensityInput.h

```diff
@@ -0,0 +1,89 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2023 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// Some code refactored from: QMCHamiltonian/EnergyDensityEstimator.h
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_ENERGY_DENSITY_INPUT_H
+#define QMCPLUSPLUS_ENERGY_DENSITY_INPUT_H
+
+#include "InputSection.h"
+#include "ReferencePointsInput.h"
+#include "SpaceGridInput.h"
+#include "type_traits/template_types.hpp"
+
+namespace qmcplusplus
+{
+namespace testing
+{
+template<typename T>
+class EnergyDensityTests;
+}
+
+class NEEnergyDensityEstimator;
```

Comment:
What does `NE` stands for?

Reply:
New Estimator, it became necessary because of ODR  and the EnergyDensityEstimator in QMCHamiltonian.
While a scheme could probably be devised around it it will require significant build changes.  It's best just not to have two identically named classes in the same applicaiton. I would like to delete the legacy estimators are rename this.